velox/functions/sparksql/MightContain.cpp

```diff
@@ -23,6 +25,11 @@
 namespace facebook::velox::functions::sparksql {
 namespace {
```

Comment:
why is this not implemented as simple function?

Reply:
The `bloomFilter_` can be different for each execution thread, but is a constant within a thread. From the document it should be registered as a stateful function. https://github.com/facebookincubator/velox/blob/6b38117f9bbe89703888d949e1121ac5e394e968/velox/docs/develop/scalar-functions.rst?plain=1#L1138-L1140

Reply:
@marin-ma I believe Laith is suggesting that this can be implemented as a simple function and bloom filter can be initialized in 'initialize' method.

See https://facebookincubator.github.io/velox/develop/scalar-functions.html#access-to-session-properties-and-constant-inputs

Reply:
@mbasmanova @laithsakka Updated the code to use SimpleFunction in the latest commit, but the function signature changed to non-const, which caused the CI to fail. Is there any way to register a const argument?
```
Found differences: {'iterable_item_added': {"root['might_contain'][0]": '(varbinary,bigint) -> boolean'}, 'iterable_item_removed': {"root['might_contain'][0]": '(constant varbinary,bigint) -> boolean'}}
 
            Incompatible changes in function signatures have been detected.
            This means your changes have modified function signatures and possibly broken backwards compatibility. 
```

Reply:
@kgpai Krishna, would you help look into this?

Reply:
@mbasmanova Should we remove the constant attribute from signatures ? I am hesitant to do that in general - Let me know though if its not required in function signatures .

Reply:
@marin-ma  Simple function interface does not support marking args as constant However its a simple change.
, would it make sense to by pass that failure (aprove the signature change) until I add that support i can ping you then to add that part?

Reply:
@laithsakka @mbasmanova In this case I think we can ignore this failure as a legitimate case. Let me know if you think otherwise.